girder/girder_large_image/web_client/vue/components/CompositeLayers.vue

```diff
@@ -216,15 +321,116 @@ export default {
     top: 0px;
     background-color: white;
     z-index: 2;
+    border-bottom: 3px solid;
+}
+.small-text {
+    font-size: 10px;
+}
+.tall-row {
+    height: 75px;
+}
+.enable-col {
+    width: 10%;
+}
+.name-col {
+    width: 40%;
+}
+.color-col {
+    width: 25%;
+}
+.auto-range-col {
+    min-width: 100px;
+    display: flex;
+    column-gap: 10px;
+    align-content: space-around;
+    padding: 0;
+}
+.auto-range-label {
+    display: flex;
+    flex-direction: column;
+}
+.switch {
+  position: relative;
+  display: inline-block;
+  width: 45px;
+  height: 20px;
+  margin-top: 5px;
+}
+.slider {
+  position: absolute;
+  cursor: pointer;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: #ccc;
+  -webkit-transition: .4s;
+  transition: .4s;
+  border-radius: 34px;
+}
+.slider.checked {
+  background-color: #2196F3;
+}
+.slider:focus{
+  box-shadow: 0 0 1px #2196F3;
+}
+.slider:before {
+  position: absolute;
+  content: "";
+  height: 15px;
+  width: 15px;
+  left: 4px;
+  bottom: 2px;
+  background-color: white;
+  -webkit-transition: .4s;
+  transition: .4s;
+  border-radius: 50%;
+}
+.slider.checked:before {
+  -webkit-transform: translateX(22px);
+  -ms-transform: translateX(22px);
+  transform: translateX(22px);
 }
 .table-container {
-    overflow-x: auto;
-    overflow-y: hidden;
+    overflow: auto;
+    max-height: 300px;
 }
 .table-container td {
     padding: 0 5px;
 }
 .table-container input {
-    max-width: 70px;
+    max-width: 80px;
+}
+tr {
+    position: relative;
+}
+.expand-btn {
+    position: absolute;
+    right: 10px;
+    top: 5px;
+}
+.advanced-section {
+    position: absolute;
+    left: 0px;
+    width: calc(100% - 10px);
+    margin: 30px 0px 0px;
+    height: 40px;
+}
+</style>
+
+<style>
+input {
```

Comment:
These need to be scoped or made more exact with some class; otherwise this affects input fields everywhere.  We probably want to add some class specific to this component.

Reply:
Maye just make the rule for `.composite-layer-table input`?

Reply:
Changed in [f2f544a](https://github.com/girder/large_image/pull/1175/commits/f2f544a9fff4295393dffb0cc1dd74da01da0808)